### DISCUSSION TITLE
Implementation of DPMSolverSinglestepScheduler

### DISCUSSION THREAD

**Discussion by wonkyoc** [huggingface/diffusers#7470] [General]
https://github.com/huggingface/diffusers/blob/a51b6cc86a5bef62283562d49497a4f3e0b134d8/src/diffusers/schedulers/scheduling_dpmsolver_singlestep.py#L637-L641

AFAIK, this implementation is multistep. The first order of the scheduler is fine but the second/third orders are not correctly implemented. Am I missing something?

**Comment by sayakpaul** [comment]
cc: @yiyixuxu
